veda/features/computation.py

```
@@ -0,0 +1,33 @@
+        """Uses WolframAlpha to compute math and conversions."""
+        try:
+            # Try to get App ID from environment variable
+            app_id = os.environ.get("WOLFRAM_APP_ID")
+
+            if not app_id:
+                # Fallback to a basic Python eval if no key is provided
+                # Only allowing very basic math chars for safety
+                import re
+                if re.match(r'^[\d\s\+\-\*\/\(\)\.]+$', query):
+                    try:
+                        result = eval(query)
```

Ruff (0.15.9)

[error] Use of possibly insecure function; consider using `ast.literal_eval`

(S307)